frontend/.storybook/preview.js

```diff
@@ -0,0 +1,4 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
```

Comment:
For my understanding, what's going on here? Maybe add a comment?

Reply:
This is actually part of the boiler plate but we're not using it so I just deleted the file